app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,25 @@
+# app/controllers/api/v1/rents_controller.rb
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_request
+      def index
+        @rents = Rent.all
+        render json: @rents
```

Comment:
We'll need to paginate every endpoint which returns an array of items, feel free to use wor-paginate again

Reply:
This wasn't clear at the ticket, so i added this comment in the original trello card